Sources/CardPayments/CardClient.swift

```diff
@@ -37,19 +37,29 @@ public class CardClient: NSObject {
         self.vaultAPI = vaultAPI
         self.webAuthenticationSession = webAuthenticationSession
     }
-    
-    @_documentation(visibility: private)
+
+    /// Updates a setup token with a payment method. Performs
+    /// 3DS verification if required. If verification is performed, SDK returns a property didAttemptThreeDSecureAuthentication.
+    /// If didAttempt3DSecureVerification is true, check verification status with /v3/vault/setup-token/{id} in your server.
+    /// - Parameters:
+    ///   - cardVaultRequest: The request containing setupTokenID and card
+    /// - Returns: Card vault result
+    /// - Throws: PayPalSDK error if vault could not complete successfully
```

Comment:
It doesn't look like this method returns or throws anything, not sure if that was missed or if these docstrings are incorrect

Reply:
I followed the pattern in approveOrder. I think it doesn't throw but returns those errors via notifyFailure => delegate function.

Reply:
Seems like we should make a ticket to update the `approveOrder` docstring then. If the results are all returned in the delegate methods, we should remove the returns/throws docs from this method and instead note that merchants need to use the delagates for result handling.

Reply:
I didn't know how to enter reply to your comment above so I'll put it down here: 
Jax: "It looks like this value is already the default in CreateSetupTokenView. We shouldn't also need to pass it at all as a default here since it's just redundant" 

Victoria: yes, if I don't set default here paymentSourceType will be initialized with verification field nil which
omits the verification_method in the setup token create call.
docs say SCA_WHEN_REQUIRED is the default, which I take to mean that in its absence, that is the value.
This was true on other endpoints, but I will ask on the vault channel just in case.

Reply:
Jax: Yes, but with both values optional in the helper method, we can pass down the appropriate data from the delegate. So the error delegate would send updateSetupTokenResponseState(vaultResult: nil, vaultError: vaultError) and the success delegate would send updateSetupTokenResponseState(vaultResult: vaultResult, vaultError: nil).

Victoria: it is actually updateUpdateSetupTokenResult if I follow the convention of referring to result of endpoint.
I will use `setUpdateSetupTokenResult`